contribution/development_guide.md

```diff
@@ -75,10 +35,60 @@
       ~/KubeArmor/KubeArmor$ make vagrant-up
       ```
 
-	    To destroy the vagrant VM
+      The ouput will come as
 
       ```text
-      ~/KubeArmor/KubeArmor$ make vagrant-destroy
+      cd /home/gourav/KubeArmor/contribution/vagrant; NETNEXT=0 DLV_RPORT=2345 vagrant up; true
```

Comment:
Put this text output in [collapsible section](https://gist.github.com/pierrejoubert73/902cc94d79424356a8d20be2b382e1ab).